velox/dwio/parquet/tests/ParquetTpchTestBase.h

```diff
@@ -73,28 +75,44 @@ class ParquetTpchTestBase : public testing::Test {
     auto task = assertQuery(tpchPlan, duckDbSql, sortingKeys);
   }
 
-  void TearDown() {
+  static void TearDownTestSuite() {
     connector::unregisterConnector(kHiveConnectorId);
-    unregisterParquetReaderFactory();
+    velox::parquet::unregisterParquetReaderFactory();
   }
 
  private:
   /// Write TPC-H tables as a Parquet file to temp directory in hive-style
   /// partition
-  void saveTpchTablesAsParquet() {
-    constexpr int kRowGroupSize = 10'000;
-    const auto tableNames = tpchBuilder_.getTableNames();
-    for (const auto& tableName : tableNames) {
+  static void saveTpchTablesAsParquet(memory::MemoryPool* pool) {
+    static constexpr auto tables = {
+        tpch::Table::TBL_PART,
+        tpch::Table::TBL_SUPPLIER,
+        tpch::Table::TBL_PARTSUPP,
+        tpch::Table::TBL_CUSTOMER,
+        tpch::Table::TBL_ORDERS,
+        tpch::Table::TBL_LINEITEM,
+        tpch::Table::TBL_NATION,
+        tpch::Table::TBL_REGION};
+
+    for (const auto& table : tables) {
+      auto tableName = std::string(toTableName(table));
       auto tableDirectory =
           fmt::format("{}/{}", tempDirectory_->path, tableName);
-      fs::create_directory(tableDirectory);
-      auto filePath = fmt::format("{}/file.parquet", tableDirectory);
-      auto query = fmt::format(
-          fmt::runtime(kDuckDbParquetWriteSQL_.at(tableName)),
-          tableName,
-          filePath,
-          kRowGroupSize);
-      duckDb_->execute(query);
+
+      auto tableSchema = velox::tpch::getTableSchema(table);
+      auto query = fmt::format("SELECT * FROM {}", tableName);
+      // TODO: The name of API `executeOrdered` is confused, it do nothing
```

Comment:
We can remove this TODO. Probably, add an issue in Velox.

Reply:
Fixed